packages/main/src/Slider.js

```diff
@@ -0,0 +1,271 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
```

Comment:
we will probably  have a slider component, the same as sap.m.Slider, so we have to agree on another name

Reply:
WheelSlider maybe? As it is in openui5?